app/decorators/legislation_decorator.rb

```diff
@@ -25,4 +29,15 @@ def litigations_links
                 title: litigation.title
     end
   end
+
+  def document_links
```

Comment:
we could extract this to `Decorators::FromDocuments` to share common `Document` - related presentation logic.